Source/ALSV4_CPP/Private/Character/ALSBaseCharacter.cpp

```diff
@@ -1284,6 +1303,13 @@ void AALSBaseCharacter::MantleUpdate(float BlendIn)
 {
 	// Step 1: Continually update the mantle target from the stored local transform to follow along with moving objects
 	MantleTarget = UALSMathLibrary::MantleComponentLocalToWorld(MantleLedgeLS);
+		
+	if (MantleLedgeLS.Component != nullptr && !IsComponentVelocityValid(MantleLedgeLS.Component, AcceptableVelocityWhileMantling))
+	{
+			//The surface to mantle moves too fast
+			MantleEnd();
```

Comment:
This line is a fallback, might not be the best way of doing it

Reply:
I'm going to merge this PR as just to check velocity on beginning of the mantle, but not during mantle action. Ending mantle during animation may cause issues.